src/portable_python/cpython.py

```diff
@@ -30,6 +43,37 @@ class Cpython(PythonBuilder):
 
     xenv_CFLAGS_NODIST = "-Wno-unused-command-line-argument"
 
+    def build_information(self):
+        """Yields key/value pairs to store as build information"""
+        yield "cpython", {
+            "prefix": self.setup.prefix,
+            "source": self.url,
+            "static": runez.joined(self.modules.selected) or None,
+            "target": PPG.target,
+            "version": self.version,
+        }
+        yield "configure-args", runez.joined(runez.short(x) for x in self.c_configure_args())
+        compiled_by = os.environ.get("PP_ORIGIN") or PPG.config.get_value("compiled-by")
+        bc = self.setup.build_context
+        yield "compilation-info", {
+            "compiled-by": compiled_by or "https://pypi.org/project/portable-python/",
+            "date": datetime.datetime.now().strftime("%Y-%m-%d %H:%M"),
```

Comment:
1. consider making this UTC for better portability `datetime.now(tz=datetime.timezone.utc).strftime("...Z")`
2. consider including seconds; I'm not sure what you gain by leaving them out

Reply:
Good point, I'll make it explicitly UTC (servers usually are, this was assuming it)
Seconds: yes actually, why not... I think originally I was thinking it's "too precise" for no good reason... but indeed, why not